hack/update_crdb_versions.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"sort"
+	"time"
+
+	"github.com/Masterminds/semver/v3"
+	"gopkg.in/yaml.v3"
+)
+
+const crdbVersionsInvertedRegexp = "^v19|^v21.1.8$|latest|ubi$"
+const crdbVersionsFileName = "crdb-versions.yaml"
+
+// TODO(rail): we may need to add pagination handling in case we pass 500 versions
+// Use anonymous API to get the list of published images from the RedHat Catalog.
+const crdbVersionsUrl = "https://catalog.redhat.com/api/containers/v1/repositories/registry/" +
+	"registry.connect.redhat.com/repository/cockroachdb/cockroach/images?" +
+	"exclude=data.repositories.comparison.advisory_rpm_mapping,data.brew," +
+	"data.cpe_ids,data.top_layer_id&page_size=500&page=0"
+const DefaultTimeout = 30
+const crdbVersionsFileDescription = `#
+# Supported CockroachDB versions.
+#
+# This file contains a list of CockroachDB versions that are supported by the
+# operator. hack/crdbversions/main.go uses this list to generate various
+# manifests.
+# Please update this file when CockroachDB releases new versions.
```

Comment:
Since you're here, would you mind adding this to the end? "Please update...versions, by running `make release/gen-templates`"

Reply:
Right next to `# Please update this file when CockroachDB releases new versions.` line?

Reply:
Or maybe even better would be something like `Generated. DO NOT EDIT. This file is created via make release/gen-templates`